components/browser/engine-system/src/test/java/mozilla/components/browser/engine/system/SystemEngineSessionTest.kt

```diff
@@ -459,74 +462,17 @@ class SystemEngineSessionTest {
         assertEquals("UTF-8", response.encoding)
     }
 
-    @Test
-    fun `shouldInterceptRequest notifies observers if request was not intercepted`() {
```

Comment:
It's not clear to me why we're removing these tests? Shouldn't they still work now, as we still call `onLoadRequest` as before (just need to remove the lambda?)

Reply:
You're correct.  I removed this earlier because these tests became not applicable when I removed the observer call from `SystemEngineView`.  But during this review it was added back in.  I should have remember to add these two tests back.  Thanks for catching this.